Home_Challenge1_Pyton.py

```diff
@@ -0,0 +1,28 @@
+file = open("C:/Users/tomer/PycharmProjects/HomeChallenge3/Readme.txt")
+words = open("C:/Users/tomer/PycharmProjects/HomeChallenge3/Readme.txt").read().split()
+
+print(file.read())
+print(words)
+
+
+def keywithmaxval(d):
```

Comment:
use '_' between word - should looks like: key_with_max_val
also, argument name should have some meaning, 'd' is not an acceptable argument name